std/file.d

```diff
@@ -3158,7 +3159,7 @@ private void copyImpl(const(char)[] f, const(char)[] t, const(FSChar)* fromz, co
                 size -= toxfer;
             }
             if (preserve)
-                cenforce(fchmod(fdw, statbuf.st_mode) == 0, f, fromz);
+                cenforce(fchmod(fdw, std.conv.to!mode_t(statbuf.st_mode)) == 0, f, fromz);
```

Comment:
[`fchmod` takes `mode_t`](https://github.com/D-Programming-Language/druntime/blob/master/src/core/sys/posix/sys/stat.d#L1032), but `st_mode` is `uint` for bionic on [`X86`](https://github.com/D-Programming-Language/druntime/blob/master/src/core/sys/posix/sys/stat.d#L941) and [`ARM`](https://github.com/D-Programming-Language/druntime/blob/master/src/core/sys/posix/sys/stat.d#L967), while [`mode_t` is `ushort` on those platforms](https://github.com/D-Programming-Language/druntime/blob/master/src/core/sys/posix/sys/types.d#L198).  So convert to `mode_t`, which shouldn't break any other platform.